align_system/algorithms/outlines_adm.py

```diff
@@ -86,6 +86,10 @@ def __init__(self,
 
             model_kwargs['torch_dtype'] = torch_dtype
 
+        if 'trust_remote_code' in kwargs:
+            if kwargs['trust_remote_code'] is True:
+                model_kwargs['trust_remote_code'] = True
+
```

Comment:
I would prefer not to "officially" support this parameter.  I will follow up with how I set these from the config